scripts/helpers.sh

```diff
@@ -112,13 +112,13 @@ function check_for_quickstart_pages_in_sitemap() {
     fi
 }
 
-function fold() {
+function group() {
     local name=$1
     local command="${@:2}"
-    echo -en "travis_fold:start:${name}\\r"
+    echo -en "::group::${name}\\r"
     echo "\$ ${command}"
     ${command}
-    echo -en "travis_fold:end:${name}\\r"
+    echo -en "::endgroup::${name}\\r"
```

Comment:
Confirmation that this works:

![Screen Shot 2022-12-09 at 6 00 42 PM](https://user-images.githubusercontent.com/72201855/206809225-3852515a-006f-4856-9fb0-16b66d7ef110.png)